src/components/calcite-color-picker/calcite-color-picker.tsx

```diff
@@ -807,7 +807,7 @@ export class CalciteColorPicker {
                   iconStart="minus"
                   label={intlDeleteColor}
                   onClick={this.deleteColor}
-                  scale={scale}
+                  scale={scale === "l" ? "m" : "s"} // TO-DO: s/m color-pickers in Figma show 16px button size (non-existent), l shows 24px size. Will this work for now (s: 24, m: 24, l: 32)?
```

Comment:
@bstifle is color-picker supposed to have 16px button size when its scale is small or medium? Or will the 24px-sized button for `s`/`m` and 32px button size for `l` suffice?

Reply:
Bryan confirmed yes, so going with this